instantsearch-image-search/react-instantsearch/src/components/debug-zone.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component, Fragment } from 'react'
+import './debug-zone.css'
+
+class DebugZone extends Component {
+  state = { hidden: true }
+
+  handleToggleClick = () => {
+    this.setState({ hidden: !this.state.hidden })
```

Comment:
```suggestion
    this.setState(state => ({ hidden: !state.hidden }))
```

Reference: https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous